pkg/localstorev2/internal/reserve/items.go

```diff
@@ -0,0 +1,264 @@
+// Copyright 2023 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package reserve
+
+import (
+	"encoding/binary"
+	"errors"
+	"path"
+
+	storage "github.com/ethersphere/bee/pkg/storagev2"
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+var errMarshalInvalidAddress = errors.New("marshal: invalid address")
+var errUnmarshalInvalidSize = errors.New("unmarshal: invalid size")
+
+type batchRadiusItem struct {
```

Comment:
Would be good to have some comments about these items and how they should be used.